source/notebooks/text_generation.ipynb

```diff
@@ -1,553 +1,528 @@
 {
```

Comment:
<p>Can you change <code># !pip install https://github.com/PyThaiNLP/pythainlp/archive/dev.zip</code> to <code># !pip install pythainlp</code><span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyThaiNLP/tutorials/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>